packages/notification-controller/src/NotificationController.test.ts

```diff
@@ -108,7 +131,9 @@ describe('NotificationController', () => {
     expect(
       // TODO: Either fix this lint violation or explain why it's necessary to ignore.
       // eslint-disable-next-line @typescript-eslint/await-thenable
-      await unrestricted.call('NotificationController:show', origin, message),
```

Comment:
Nit: Can we remove the await and remove the TODO ?
```suggestion
      unrestricted.call('NotificationController:show', origin, message),
```